ml-agents/mlagents/trainers/torch/components/reward_providers/gail_reward_provider.py

```diff
@@ -69,31 +69,23 @@ def __init__(self, specs: BehaviorSpec, settings: GAILSettings) -> None:
         self._use_vail = settings.use_vail
         self._settings = settings
 
-        state_encoder_settings = NetworkSettings(
+        encoder_settings = NetworkSettings(
             normalize=False,
             hidden_units=settings.encoding_size,
             num_layers=2,
```

Comment:
Should we reduce the number of layers here since there is an additional hidden layer in the self._estimator ?